### PR TITLE
10 create index page and css

### PR DESCRIPTION
**Pull request by AnaisBonin**
- Created index page with first title and introdution
- Created CSS stylesheet with:
- - boilerplate (removed auto margin, decoration, colors for links...)
- - variantes for colors
- - fonts

### CODE REVIEW

**Review by baptiste-mrtn — APPROVED** [review]
tro bien wola c bo

**Comment by debbabi** [comment]
Le nom de la branche ne respecte pas la convention! feature/...

**Comment by XavierFavier** [comment]
Je suis d'accord avec debbabi

**Comment by debbabi** [comment]
Can you add a placeholder to the different navigation links?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <body class="homepage">
+    <header class="home-header">
+      <h1 class="home-title">Hi everybody</h1>
+    </header>
+    <main class="home-main">
+      <h2>Our broken champions:</h2>
+      <p>
+        Please give a warm welcome to everyone and give Anais a lot of money
+      </p>
+    </main>
```
Comment (debbabi) [inline index.html:29]: Can you add a placeholder to the different navigation links?

**Review by debbabi — CHANGES_REQUESTED** [review]

**Comment by boubacargadio** [comment]
in english please

**Review by nicolasnieloux — APPROVED** [review]

**Review by IreneuszPISKORSKI — APPROVED** [review]

**Review by Narmine38 — APPROVED** [review]
ok for me

**Review by CharlyLstudent — APPROVED** [review]

**Pull request closed without merging**